.github/workflows/debezium-workflow.yml

```
@@ -152,12 +153,23 @@ jobs:
             pom.xml
             .github/workflows/debezium-workflow.yml
 
-      - name: Get modified files (DDL parser)
-        id: changed-files-ddl-parser
+      - name: Get modified files (MySQL DDL parser)
+        id: changed-files-mysql-ddl-parser
         uses: tj-actions/changed-files@v14.3
         with:
           files: |
-            debezium-ddl-parser/**
+            debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/**
+            debezium-ddl-parser/src/main/java/io/debezium/antlr/**
+            debezium-ddl-parser/src/test/resources/mysql/examples/**
+
+      - name: Get modified files (Oracle DDL parser)
+        id: changed-files-oracle-ddl-parser
+        uses: tj-actions/changed-files@v14.3
+        with:
+          files: |
+            debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/**
+            debezium-ddl-parser/src/main/java/io/debezium/ddl/parser/oracle/**
```

```suggestion
            debezium-ddl-parser/src/main/java/io/debezium/antlr/**
            debezium-ddl-parser/src/main/java/io/debezium/ddl/parser/oracle/**
```
For Oracle, we also need to include the files under the `io.debezium.antlr` package as the `OracleDdlParser` is also based on these files as well as the MySQL DDL parser implementation.